src/services/OpenKlantRegistrationService/Listener/ZakenApi.ts

```diff
@@ -53,6 +53,7 @@ export class ZakenApi extends ZgwApi implements IZakenApi {
       const result = await response.json();
       return RolSchema.parse(result);
     } catch (error) {
+      console.error('ROL UPDATE FAILED'); // This is picked up by a critical level alarm
       console.error('Failed to delete and recreate rol! This is the original role:', originalRol);
```

Comment:
Moet hier nog anonimiseren bij?

Reply:
Nee want de rol wordt gedumpt in de logging in zijn gehele (dit hebbene we nodig als we een foute update moeten herstellen). Wel moeten we zorgen dat er een KMS key op de log groups komt. Ik maak een issue aan. Voor nu geen probleem omdat er nog geen vul service in productie staat.